### ISSUE TITLE
Cannot Change Multi-Farm Type From Default

### ISSUE THREAD

**Issue by wash4717** [ForestryMC/ForestryMC#643]
MC Rel.  1.7.10 Forge 10.13.2.1291.  Cannot insert properly programmed board into circuit board slot when trying to change / program farm type and allowed objects, crops, saplings, vegetables, etc.
Managed farm works fine with default for trees and saplings.  ( With soldering iron gui )
The circuit board just flies back into inventory.  I have tried everything I can think of, combinations of process, searched web and forums forever, to no avail.
Would greatly appreciate any help.  I love this Mod and want to use all it's functionality.
Thanks for any help.

Glen Washburn   wash4717@gmail.com

**Comment by Nirek-K** [comment]
To properly program a circuit board, you open the soldering iron (rightclick in hand)
Place the electron tubes in the gui first, you should see what type of farm that slot will become.
(change the arrows on the top to switch between managed and manual farm types)
Once all 4 tubes are in place, put an Intricate (important, only intricate) in the top left slot of the gui.
The completed board will appear in the lower right slot. Place the completed board in the circuit slot of the farm ( the center of 4 tree icons on the default farm gui).

**Comment by wash4717** [comment]
Hi;

I really appreciate your quick response, but I have done exactly as you
suggest, along with many other combinations, etc.  The circuit board
programs correctly, no issues, but when finished, it will not go into the
empty circuit board slot, in the position you indicated.

I have tried all four board types with appropriate tubes, etc, to no
avail.  I have used several older versions of the forestry mod also.

There was another individual I just read about who had the same issue, and
someone suggested he place the circuit board a little higher, at the top of
the empty socket slot, does not work anywhere for me.

Very frustrating.  I have been fooling around with this for days.  I love
the mod and want to use the multi farm mod.  I can get the
default farm to work fine without any circuit board, grow all kinds of
trees, produce wood block, empty ash, everything seem to work fine.

Am stumped.

Again: Thank you so much for your response.

Have a nice evening.

Glen Washburn

**Comment by wash4717** [comment]
I really appreciate your quick response, but I have done exactly as you suggest, along with many other combinations, etc.  The circuit board programs correctly, no issues, but when finished, it will not go into the empty circuit board slot, in the position you indicated.

I have tried all four board types with appropriate tubes, etc, to no avail.  I have used several older versions of the forestry mod also.

There was another individual I just read about who had the same issue, and someone suggested he place the circuit board a little higher, at the top of the empty socket slot, does not work anywhere for me.

Very frustrating.  I have been fooling around with this for days.  I love the mod and want to use the multi farm function.  I can get the default farm to work fine without any circuit board, grow all kinds of trees, produce wood block, empty ash, everything seems to work fine.

Am stumped.

Thank you again.

**Comment by Nirek-K** [comment]
Other than updating to current versions, Forge 1391, Forestry 3.5.4,
I'm not sure what to tell you, they work correctly functionally in my testing on current.

**Comment by wash4717** [comment]
Hi;

Thank you again.  I am running on that version of Forestry, but not Forge.

Have a nice evening.

**Comment by wash4717** [comment]
Update:

I tried installing the Forestry Mod  Rel. 1.7.10-3.5.4.13, Forge Rel. 10.13.2.1291 in an earlier version of Minecraft I had saved, with no other mods installed.  It worked fine, but only allowing me to use three electron tubes with the managed crop farm, but it worked.  So, I believe I have some type of mod  / rel. incompatibility issue.  But now I know it does work with some  combination, I can further test and eliminate issues.

Thanks for your time and help.
